apps/src/code-studio/assets/SpriteUpload.js

```diff
@@ -22,20 +57,30 @@ export default class SpriteUpload extends React.Component {
         <h1>Sprite Upload</h1>
         <form onSubmit={this.handleSubmit}>
           <label>
-            <h4> Sprite Category: </h4>
-            <input type="text" />
+            <h4> Sprite Category:</h4>
```

Comment:
nit: there are still spaces after `<h4>` for these elements

Reply:
:facepalm: Good catch. Updating